components/Extension/helper.tsx

```diff
@@ -0,0 +1,25 @@
+import { PostContent } from "@subsocial/api/types"
+import { registry } from "./registry";
+import { Extension, IExtension } from "./types";
+import { EmptyExtension } from "./emptyExtension";
+
+function getRegisteredExtension(extension: Extension): IExtension | null {
+  const index = registry.findIndex((v) => v.schemaName == extension.type)
+  if (index == -1) new EmptyExtension();
```

Comment:
forgot to add return statement, purpose of this empty extension is to use it as a template for creating new extension by copying the file. 

And also it will be shown in case of the schema type fetched from IPFS doesn't exists in the registry. So it can show message like: "Message contains chat extensions that can't be loaded"